build.gradle

```diff
@@ -86,7 +86,7 @@ apply from: "${rootDir}/scripts/version-updater.gradle"
 ext {
 //    gliaSdkVersion = 'X.X.X' // this property is now attached dynamically during gradle sync
 
-  appCompatVersion = '1.3.1'
+  appCompatVersion = '1.6.1'
```

Comment:
@Pelkar Is it possible to update it in your [PR](https://github.com/salemove/android-sdk-widgets/pull/1006/files)?
@DavDo Will your changes work well if you keep the old version?

Reply:
If we move this change to Karls's PR, we will avoid merge conflicts.

Reply:
I will keep [feature/MOB-3207-improve-notification-experience](https://github.com/salemove/android-sdk-widgets/tree/feature/MOB-3207-improve-notification-experience) unmerged until I have finished the AT alignment. And since I will rebase it on top of the development branch to keep it fresh, I will do that during rebase.